keps/sig-node/1769-memory-manager/README.md

```diff
@@ -521,54 +586,61 @@ _This section must be completed when targeting alpha to a release._
     - Will enabling / disabling the feature require downtime or reprovisioning
       of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled). Yes, it uses a feature gate.
 
-* **Does enabling the feature change any default behavior?**
+###### Does enabling the feature change any default behavior?
+
 Yes, the admission flow changes for a pod in Guaranteed QoS class. With the Memory Manager, Topology Manager also takes into account allocatable memory (and hugepages) to either admit a pod to the node or reject it. 
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?** Yes, it uses a feature gate.
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes, it uses a feature gate.
 
   <!-- Also set `disable-supported` to `true` or `false` in `kep.yaml`.
   Describe the consequences on existing workloads (e.g., if this is a runtime
   feature, can it break the existing applications?). -->
 
-* **What happens if we reenable the feature if it was previously rolled back?**
+###### What happens if we reenable the feature if it was previously rolled back?
+
 The Memory Manager utilizes the state file to track memory assignments. If State file is not valid, it must be removed and kubelet restarted. E.g., State file might become invalid when kube/system reserved have changed (increased), which may lead to a situation when some containers cannot be started.
 
-* **Are there any tests for feature enablement/disablement?**
+###### Are there any tests for feature enablement/disablement?
```

Comment:
The answer below doesn't seem to answer the question, can we double check if we have tests (unit/e2e) where the feature enablement and disablement is tested?

Reply:
I'm wondering if this is still relevant for GA'ing. If so I'd be happy to address this point.